### PR TITLE
Question page

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Added the question pages to the quiz app. Please review my PR.

Preview: https://unruffled-austin-a7f740.netlify.app/pages/questionpage/basicgardening/question1

### CODE REVIEW

File: Pages/questionPage/basicGardening/question1.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <p>Made with <i class="red-color fas fa-heart"></i> by Vrinda Mahajan </p>
+        <ul class="m2 list-non-bullet">
+            <li class="p1 inline"><a class="footer-icons" target="_blank" href="https://github.com/vrinda-mahajan"><i class="fab fa-github"></i></a></li>
+            <li class="p1 inline"><a class="footer-icons" target="_blank" href="https://twitter.com/vrinda_mahajan"><i class="fab fa-twitter"></i></a></li>
+            <li class="p1 inline"><a class="footer-icons" target="_blank" href="https://www.linkedin.com/in/vrinda-mahajan-7a2b2a20b/"><i class="fab fa-linkedin-in"></i></a></li>
+        </ul>
+        <div>© copyright 2022 Flora UI</div>
+    </footer>
+</body>
+</html>
```
Comment (jalaj-gl) [inline Pages/questionPage/basicGardening/question1.html:57]: just a nitpick but adding a newline at the end of files is considered a good practice
https://thoughtbot.com/blog/no-newline-at-end-of-file

**Review by jalaj-gl — COMMENTED** [review]
-

File: Pages/questionPage/basicGardening/basicGardening.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    padding: 1.6rem;
+}
+
+.question-option:hover {
+    background-color: var(--option-bg-color-hover);
+}
+
+.question-option:focus {
+    background-color: var(--primary-color2);
+}
```
Comment (jalajcodes) [inline Pages/questionPage/basicGardening/basicGardening.css]: just a nitpick but adding a newline at the end of files is considered a good practice
https://thoughtbot.com/blog/no-newline-at-end-of-file

**Review by jalajcodes — APPROVED** [review]
Everything else look good! 

Also you can dismiss my above comment (by `jalaj-gl`), forgot to switch accounts😅

File: Pages/questionPage/basicGardening/basicGardening.css (hunk 2 above)
Comment (vrinda-mahajan) [reply Pages/questionPage/basicGardening/basicGardening.css]: Thanks for pointing that out👍

**Pull request merged**